Documentation/project-docs/conferences.md

```diff
@@ -0,0 +1,47 @@
+## .NET Conferences and Announcements links
+
+### .NET Core 3.0
+
+* [.NET Core 3.0 previews](https://github.com/dotnet/core/tree/master/release-notes/3.0)
```

Comment:
Information about 3.0 previews is irrelevant now that 3.0 shipped. 

This whole page looks very questionable to me. Is it really useful?

Reply:
Information on this page is the purpose of the dotnet/core repo.  I would just delete this whole .md file.